doc/man7/openssl-glossary.pod

```diff
@@ -69,18 +69,19 @@ external format such as PEM or DER.
 
 L<OSSL_ENCODER_CTX_new_for_pkey(3)>
 
+=item Explicit Fetching
+
+Explicit Fetching is a type of Fetching (see Fetching). Explicit Fetching is
+where a function call is made to obtain an algorithm object representing an
+implementation such as L<EVP_MD_fetch(3)> or L<EVP_CIPHER_fetch(3)>
+
 =item Fetching
 
 Fetching is the process of looking through the available algorithm
 implementations, applying selection criteria (via a property query string), and
-finally choosing the implementation that will be used. Fetches can either be
-explicit where a function call is made to obtain an algorithm object
-representing an implementation such as L<EVP_MD_fetch(3)> or
-L<EVP_CIPHER_fetch(3)>, or implicit where an algorithm object with no associated
-implementation is used such as the return value from L<EVP_sha256(3)> or
-L<EVP_aes_128_cbc(3)>. With implicit fetching an implementation is fetched
-automatically using default selection criteria the first time the algorithm is
-used.
+finally choosing the implementation that will be used.
+
+Also see Explicit Fetching and Implict Fetching.
```

Comment:
Link?

Reply:
`=item` aren't anchors...

Reply:
As per @levitte's comment, its not actually possible since they aren't anchors.